warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -596,6 +598,7 @@ public void run() {
       connector.setIdleTimeout(idleTimeout);
       connector.setPort(port);
       connector.setHost(host);
+      connector.setAcceptQueueSize(tcpBacklog);
```

Comment:
The same logic should be added for the SSL endpoints and for the standlone HTTP and HTTPS endpoints.
Adding the same logic for the Plasma endpoints (for both distributed and standalone versions) should be considered.